### DISCUSSION TITLE
zh-cn/posts/2024/07/09/hugo-customize-tag-title/

### DISCUSSION THREAD

**Discussion by giscus** [CyrusYip/blog-comments#76] [Comments]
# zh-cn/posts/2024/07/09/hugo-customize-tag-title/

背景 # 一开始我用 Hugo 做中文网站时打算在文章里用中文标签，但是链接里面的中文会被转换为人类读不懂的字符。比如：/zh-cn/tags/博客/ 会变成 /zh-cn/tags/%E5%8D%9A%E5%AE%A

https://cyrusyip.org/zh-cn/posts/2024/07/09/hugo-customize-tag-title/

**Comment by chuxinyuan** [comment]
哈哈，很少有人注意这个问题，或者注意到可能也不知道怎么解决。
